AirCasting/CreateSessionViews/SelectPeripheralView.swift

```diff
@@ -95,7 +96,7 @@ struct SelectPeripheralView: View {
     
     var titleLabel: some View {
         var title: Text
-        if syncMode == true {
+        if syncMode == true || SDClearingRouteProcess {
```

Comment:
I realise it's mine, but could you please delete this "== true"?

Reply:
As long as syncMode is optional, unfortunately I cannot do it